hyperspy/_signals/eels.py

```diff
@@ -751,6 +751,111 @@ def fourier_ratio_deconvolution(self, ll,
                 self.tmp_parameters.filename +
                 'after_fourier_ratio_deconvolution')
         return cl
+    
+    def fourier_ratio_deconvolution_llspectra(self, kernel, zl=None,
```

Comment:
zl should be zlp, which is the common notation for the zero loss peak in the rest of hyperspy